Task-7294: Clean old wiki code

mkrout opened the pull request.

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Merge branch 'develop' into clean_code

houssemexo26 committed:
Merge branch 'develop' into clean_code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Merge remote-tracking branch 'origin/clean_code' into clean_code

mkrout committed:
Merge branch 'develop' into clean_code

boubaker commented on notes-service/src/main/java/org/exoplatform/wiki/jpa/EntityConverter.java:
```diff
@@ -1,3 +1,22 @@
+/*
+ * This file is part of the Meeds project (https://meeds.io/).
+ *
+ * Copyright (C) 2020 - 2021 Meeds Association contact@meeds.io
```
```suggestion
 * Copyright (C) 2020 - 2022 Meeds Association contact@meeds.io
```
same for other files please

boubaker commented on notes-service/pom.xml:
```diff
@@ -11,7 +11,7 @@
   <properties>
-    <exo.test.coverage.ratio>0.44</exo.test.coverage.ratio>
+    <exo.test.coverage.ratio>0.42</exo.test.coverage.ratio>
   </properties>
```
please cleanup the dependencies of this artifact to keep only used ones, such as https://github.com/Meeds-io/notes/blob/7d4362d9ce5cc57336b77bed1b8c3813f5dfd263/notes-service/pom.xml#L84-L105

mkrout replied:
Done

boubaker commented on notes-service/src/main/java/org/exoplatform/wiki/model/DraftPage.java:
```diff
@@ -0,0 +1,39 @@
+
+  private boolean newPage;
+
+  private DiffResult changes;
```
overall code format

boubaker commented on notes-service/src/main/java/org/exoplatform/wiki/model/ImportList.java:
```diff
@@ -0,0 +1,46 @@
+  }
+
+
+
```
multiple spaces

boubaker commented on notes-service/src/main/java/org/exoplatform/wiki/model/PageHistory.java:
```diff
@@ -0,0 +1,39 @@
+@Data
+public class PageHistory {
+
+	private Long versionNumber;
```
please avoid using tabs (overall code format)

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Merge branch 'develop' into clean_code

mkrout committed:
Merge branch 'develop' into clean_code

boubaker commented on notes-packaging/src/main/assemblies/notes-addon-package.xml:
```diff
@@ -42,24 +42,8 @@
       <includes>
         <include>${project.groupId}:*:jar</include>
         <include>commons-configuration:commons-configuration:jar</include>
```
![image](https://user-images.githubusercontent.com/594824/170483542-0f3edcf8-0149-4479-bec9-67212433a854.png)
apparently this dependency isn't included, same as for (net.sf.json-lib) https://github.com/Meeds-io/notes/pull/540/files#diff-b97cf76091695f7a02119d50c025ed92c53613d4368e8d86502825052dae5899R46

boubaker commented on notes-service/src/main/java/org/exoplatform/wiki/notification/listener/EditWikiListener.java:
```diff
@@ -55,7 +55,7 @@ private NotificationContext buildContext(WikiService wikiService, Page page) {
 
     // Task creator
-    try {
+    /*try {
```
please delete commented code (DEVSTD)

boubaker commented on notes-service/src/main/java/org/exoplatform/wiki/service/NoteService.java:
```diff
@@ -359,9 +361,42 @@ DraftPage createDraftForExistPage(DraftPage draftNoteToSave,
-  void importNotes(String zipLocation, Page parent, String conflict, Identity userIdentity) throws WikiException, IllegalAccessException, IOException;
+    String getNoteRenderedContent(Page note);
+
+    void importNotes(String zipLocation, Page parent, String conflict, Identity userIdentity) throws WikiException, IllegalAccessException, IOException;
```
code format

boubaker commented on notes-service/src/test/java/org/exoplatform/wiki/service/TestWikiService.java:
```diff
@@ -37,7 +42,11 @@
+
+
+
+TestWikiService extends BaseTest {
```
code format

boubaker commented on pom.xml:
```diff
@@ -16,7 +16,6 @@
   <modules>
     <module>notes-service</module>
     <module>notes-webapp</module>
-    <module>notes-webui</module>
```
please delete the dependency management entry as well
```
      <dependency>
        <groupId>${project.groupId}</groupId>
        <artifactId>notes-webui</artifactId>
        <version>${project.version}</version>
      </dependency>
```

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Merge remote-tracking branch 'origin/clean_code' into clean_code

boubaker committed:
TASK-56997 Avoid double statistics collection for a Note Bookmark (#552)

Prior to this change, when bookmarking a Note Page, two entries are added in collected statistics about user action. This is due to the fact that the Favorite Button has already an automatic statistic collection. Thus we should disable the specific statistic collection added for a Note Bookmarking.
This change will delete introduced listener to add a statistic collection entry dedicated to bookmarking a Note page.

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Task-7294: Clean old wiki code

mkrout committed:
Merge remote-tracking branch 'origin/clean_code' into clean_code

# Conflicts:
#	notes-social-integration/src/main/java/org/exoplatform/notes/listener/analytics/NotesMetadataListener.java
